content/installation/ocaml.md

```diff
@@ -6,4 +6,49 @@ implementation: official
 weight: 1125
 ---
 
-Please see the [Cucumber OCaml website](https://github.com/cucumber/cucumber.ml).
+[Cucumber.ml](https://github.com/cucumber/cucumber.ml) is an
+implementation written in and useful for
+[OCaml](http://www.ocaml.org/) projects.
+
+# Prerequisites
+
+There are a few complications when working with
```

Comment:
* Consider switching the order here: Mention the prerequisite first, and then clarify below as to why. 
* If possible/needed, provide short steps on how to meet these prerequisites and/or check that they have been met (to prevent trouble shooting later on). In general, prevent lengthy text. (For an example of using bullit point, have a look at the 10 minute tutorial also).
* If possible, show the terminal commands inside code blocks marked as shell
* You mention Linux explicitly. Are instruction the same / different for Windows / MacOS? If relevant, could you add those?

Reply:
Thanks! This is exactly what I was looking for.